install_bootstrap

```diff
@@ -314,15 +314,25 @@ while [[ "${run_boardwalk^^}" != 'Y' &&  "${run_boardwalk^^}" != 'N' ]] ; do
         google_site_verification_code='google_site_verification_code'
         ask_question "What is your Google site verification code, something like tk74JOp1VBqdRQVTirM8WKpa1VIYW_vwxfoJo53inH8?" "$GOOGLE_SITE_VERIFICATION_CODE" "Google Site Verification Code" $google_site_verification_code
 
+	# bagit-firecloud-lambda 
```

Comment:
I know we discussed this and you disagree, but I think you should get rid of the bagit stuff in this PR. This PR finalizes the completion of the presigned url, and it's weird to be prompted for the Bagit info when it's not being used.

Reply:
Removed all variables referring to the `bagit-firecloud-lambda`, brought the docker compose system down, then up again, and tested the endpoint. I did not go through the installation process.